README.md

```diff
@@ -187,4 +200,4 @@ Note that in the [doc/books](https://github.com/pygae/galgebra/blob/master/doc/b
 - `galgebra.pdf` which is the original main doc of GAlgebra in PDF format, while the math part is still valid, the part describing the installation and usage of GAlgebra is outdated, please read with cautious or visit https://galgebra.readthedocs.io/ instead.
 - `Macdonald` which constains bundled supplementary materials for [Linear and Geometric Algebra](http://www.faculty.luther.edu/~macdonal/laga/index.html) and [Vector and Geometric Calculus](http://www.faculty.luther.edu/~macdonal/vagc/index.html) by Alan Macdonald, see [here](https://github.com/pygae/galgebra/blob/master/doc/books/Macdonald/) and [here](https://github.com/pygae/galgebra/blob/master/examples/Macdonald/) for more information.
 
-[brombo/galgebra]: https://web.archive.org/web/20180611145113/https://github.com/brombo/galgebra
```

Comment:
This line had to go, because breaking the md file up into parts caused it to be lost.

Now that brombo/galgebra is "alive" again, we don't need it anyway.